reana_db/models.py

```diff
@@ -290,22 +290,66 @@ class JobStatus(enum.Enum):
     queued = 5
 
 
+class WorkflowSession(Base):
+    """Workflow Session table."""
+
+    __tablename__ = "workflow_session"
+    __table_args__ = {"schema": "__reana"}
+
+    workflow_id = Column(UUIDType, ForeignKey("__reana.workflow.id_"), nullable=True)
+    session_id = Column(
+        UUIDType, ForeignKey("__reana.interactive_session.id_"), primary_key=True
+    )
+
+    def __repr__(self):
+        """Workflow Session string representation."""
+        return "<WorkflowSession {} {}>".format(self.session_id, self.workflow_id)
+
+
+class InteractiveSessionType(enum.Enum):
+    """Enumeration of interactive session types."""
+
+    jupyter = 0
+
+
+class InteractiveSession(Base, Timestamp):
+    """Interactive Session table."""
+
+    __tablename__ = "interactive_session"
+    id_ = Column(UUIDType, primary_key=True, unique=True, default=generate_uuid)
+    name = Column(String(255))
+    path = Column(Text)  # path to access the interactive session
+    status = Column(Enum(RunStatus), nullable=False, default=RunStatus.created)
+    owner_id = Column(UUIDType, ForeignKey("__reana.user_.id_"))
+    type_ = Column(
+        Enum(InteractiveSessionType),
+        nullable=False,
+        default=InteractiveSessionType.jupyter,
+    )
+
+    __table_args__ = (
+        UniqueConstraint("name", "path", name="_interactive_session_uc"),
+        {"schema": "__reana"},
+    )
+
+    def __repr__(self):
+        """Interactive Session string represetantion."""
```

Comment:
```suggestion
        """Interactive Session string representation."""
```

Reply:
Nice! You have a good eyes! I've actually copied it from Workflow model, so the issue is there too (and also Users model :D)

Reply:
Definitely a sharp eye for detail, thanks Marco 🤗